### DISCUSSION TITLE
Adding a Napalm Driver

### DISCUSSION THREAD

**Discussion by imkwx** [netbox-community/netbox-docker#380] [Getting Help] [answered]
Hello community!

Today I wanted to try out the napalm integration. I was successful for many of our devices (mainly Cisco devices) but when I tried for our fortigate devices I ended up having an error thrown saying that the napalm driver fortios wasn't found.

I took a look at the dockerfile and found out that only napalm was installed but not the backend library pyFG. Can it be added upstream or is there a solution so that I can add it for my installation?

Have a good day and thanks :D

**Comment by cimnine** [comment, marked as answer]
You can enhance the image yourself. It's briefly documented here: https://github.com/netbox-community/netbox-docker/wiki/Build#enhance-the-image

Probably it's something along the lines of the following example, though I have not tested this:

```Dockerfile
FROM netboxcommunity/netbox:v2.x.x # change to the version you're using

RUN pip install pyFG
```

**Reply by imkwx** [reply]
Thank you so much for your answer,

I'll try that ASAP !

Have a good day!

**Reply by FlorianHeigl** [reply]
Thank you - i had the same question!
